modules/migrations/github.go

```diff
@@ -443,10 +462,10 @@ func (g *GithubDownloaderV3) GetPullRequests(page, perPage int) ([]*base.PullReq
 			MergedTime:     pr.MergedAt,
 			IsLocked:       pr.ActiveLockReason != nil,
 			Head: base.PullRequestBranch{
-				Ref:       *pr.Head.Ref,
-				SHA:       *pr.Head.SHA,
+				Ref:       headRef,
```

Comment:
Will base be able to cope with these being effectively empty?

Reply:
Base Repo's `Ref`, `SHA` information should be stored in databased even if the branch deleted. So they will not be empty, but it will fail on coming process.